base/compiler/ssair/ir.jl

```diff
@@ -52,8 +52,34 @@ struct CFG
     blocks::Vector{BasicBlock}
     index::Vector{Int} # map from instruction => basic-block number
                        # TODO: make this O(1) instead of O(log(n_blocks))?
+    domtree
```

Comment:
More concrete type on this field?

Reply:
Right, I forgot to mention--domtree.jl is `include`d after ir.jl, but uses `CFG` and `BasicBlock`, which are defined in ir.jl, so we'll have to do some reorganizing for this to work.